ui/src/app/shared/components/layout/layout.tsx

```diff
@@ -12,10 +12,13 @@ export interface LayoutProps {
     isExtension?: boolean;
 }
 
+const getBGColor = (theme: string): string => (theme === 'light' ? '#dee6eb' : '#100f0f');
+
 export const Layout = (props: LayoutProps) => (
     <div className={props.pref.theme ? 'theme-' + props.pref.theme : 'theme-light'}>
         <div className={`cd-layout ${props.isExtension ? 'cd-layout--extension' : ''}`}>
             <Sidebar onVersionClick={props.onVersionClick} navItems={props.navItems} pref={props.pref} />
+            {props.pref.theme ? (document.body.style.background = getBGColor(props.pref.theme)) : null}
```

Comment:
@saumeya But this will output the color to the body and break the layout.

![image](https://github.com/argoproj/argo-cd/assets/1651867/ef0cffbe-a77b-4d1e-9416-4c14e6718669)

![image](https://github.com/argoproj/argo-cd/assets/1651867/b4700360-1bc7-4cff-992c-d4c068a1b6fa)